### DISCUSSION TITLE
Add Ctrl+Z support for textboxes

### DISCUSSION THREAD

**Discussion by andrewhong04** [ppy/osu#24205] [Ideas]
osu!lazer currently doesn't support this.

**Comment by nagi-desuuu** [comment]
What's the use case for this?

**Reply by andrewhong04** [reply]
Undo changes in a textbox...

**Reply by peppy** [reply]
I think what is being asked here is: What textbox would you be looking to do this in? It's non-trivial to add undo support in textboxes and we wouldn't be considering it unless there's a valid case.
